airflow/providers/google/cloud/transfers/gcs_to_gcs.py

```diff
@@ -313,6 +318,14 @@ def _copy_source_without_wildcard(self, hook, prefix):
                 self._copy_single_object(
                     hook=hook, source_object=prefix, destination_object=self.destination_object
                 )
+            else:
+                msg = (
+                    f'{prefix} does not exist in bucket {self.source_bucket}'
+                )
+                self.log.warning(msg)
```

Comment:
maybe move the log inside the if, so current users do not log with WARNING level